### PR TITLE
dug_6_osfv: add

### PR DESCRIPTION
**Pull request by macpijan**

### CODE REVIEW

File: dug_6_osfv.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+---
+
+# Current state
+
+* No release since the last presentation
+* Still quite intense development in the `develop` branch
+  - all improvements target this branch now
+  - if you want to experiment, this should be a starting point
+  - we are aiming for something "stable enough" to merge into `main` and
+    release v0.3.0 version
```
Comment (pietrushnic) [inline dug_6_osfv.md:31]: let's target DUG#7

File: dug_6_osfv.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+.center[<img src="/remark-templates/dasharo-presentation-template/images/dasharo-sygnet-white.svg" width="150px" style="margin-left:-20px">]
+
+???
+
+---
+
+# TBD: Agenda
+
+* Introduction to Dasharo Open Source Firmware Validation (OSFV)
```
Comment (pietrushnic) [inline dug_6_osfv.md:13]: I would use intro from last presentation and try to improve it, so it better reflects current state of the art or maybe even little bit show vision where we going with usage of OSFV and its interaction with lab infrstructure.

File: dug_6_osfv.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  - more details in DCU presentation
+* Generate documentation from test code
+  - host on github pages for starters
+* osfv_cli integration
+  - integrate low-level hardware operations into Python libraries
+  - avoid doing that in RF, it is often tedious
+
+---
+
+# osfv_cli
```
Comment (pietrushnic) [inline dug_6_osfv.md:130]: This one deserves more attention; it is a magnificent tool that is very useful while doing remote work.

File: dug_6_osfv.md (hunk 2 above)
Comment (pietrushnic) [reply dug_6_osfv.md:13]: Let's try to present various users perspective:
- QEMU validation of DTS
- dev use cases - usage while adding regression tests, test driven bug fixing, usage while developing, code rebase verification
- val use cases - regression testing, release testing

File: dug_6_osfv.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+* Recent improvements
+* Getting started with OSFV
+* Upcoming Features: OSFV v0.3 and Beyond
+* Q&A
+
+???
+
+---
+
+# Current state
```
Comment (pietrushnic) [inline dug_6_osfv.md:24]: I miss stats, how many tests we have, what are we cover?

File: dug_6_osfv.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+---
+
+# In progress - current priorities
+
+* DCU integration
+  - alternative interface for changing firmware settings
+  - instead of manual steps via PiKVM / serial, we can modify SMMSTORE variables directly
+  - more details in DCU presentation
+* Generate documentation from test code
+  - host on github pages for starters
```
Comment (pietrushnic) [inline dug_6_osfv.md:123]: link?

File: dug_6_osfv.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+    + NovaCustom
+    + Protectli
+    + PC Engines
+    + QEMU
+    + MSI
+
+---
+class: center, middle, intro
+
+# Q&A
```
Comment (pietrushnic) [inline dug_6_osfv.md:152]: Missing call to action: What can the community do for us?

**Review by pietrushnic — CHANGES_REQUESTED** [review]

File: dug_6_osfv.md (hunk 3 above)
Comment (macpijan) [reply dug_6_osfv.md]: cbc21481e3640464bef54f2e6fc2468170e596b8

File: dug_6_osfv.md (hunk 1 above)
Comment (macpijan) [reply dug_6_osfv.md:31]: 1d60196c1fe7e376ea90d709fdfff9edee19c930

File: dug_6_osfv.md (hunk 5 above)
Comment (macpijan) [reply dug_6_osfv.md]: there is none yet, added image cbd8a35bc643b65ca99e073ecc76f05d71b6d3a6

File: dug_6_osfv.md (hunk 4 above)
Comment (macpijan) [reply dug_6_osfv.md:24]: added stats: fdff2448bf004b884f20796ce4d58ae8c0a47eac

**Comment by macpijan** [comment]
I think pre-commit rules need adjustment. For 3-levels bullet points, we need 4 spaces indentation to render correctly, while it fixes to 2.

**Pull request merged**